packages/typespec-ts/test/commands/cadl-ranch-list.js

```diff
@@ -81,7 +82,7 @@ export const rlcTsps = [
   },
   {
     outputPath: "lro/lroRPC",
-    inputPath: "azure/core/lro/rpc-legacy"
+    inputPath: "azure/core/lro/rpc"
```

Comment:
can we change the output path to be consistent with modular?

Reply:
so let me revisit this.